SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaDatabase.h

```diff
@@ -2,6 +2,7 @@
 
 #pragma once
 
+#include "Containers/Map.h"
```

Comment:
Schema database version needs to be updated

Reply:
good point